state-chain/runtime/src/chainflip.rs

```diff
@@ -164,7 +164,11 @@ impl EvmPriorityFee<Ethereum> for EthTransactionBuilder {
 		Some(U256::from(tracked_data.priority_fee))
 	}
 }
-impl EvmPriorityFee<Arbitrum> for ArbTransactionBuilder {}
+impl EvmPriorityFee<Arbitrum> for ArbTransactionBuilder {
+	fn get_priority_fee(tracked_data: &<Arbitrum as Chain>::TrackedData) -> Option<U256> {
+		Some(U256::from(0))
```

Comment:
Setting this to zero instead of None because otherwise the CFE ends up estimating and setting a priority fee as part of the `estimate_gas` call`. We want it to be zero.

Reply:
Maybe add a comment to this effect so nobody feels tempted to change it.